firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1767,6 +1767,11 @@ extension String {
                 tableName: nil,
                 value: "Use your fingerprint to access passwords now.",
                 comment: "Touch ID prompt subtitle when accessing logins and passwords")
+            public static let UseSavedLoginFromKeyboard = MZLocalizedString(
+                key: "", // Settings.Passwords.Autofill.UseSavedLoginFromKeyboard.v124
```

Comment:
why commented?

Reply:
It's trick we used for Fakespot when the design and strings changed to avoid being exported until the feature was completed.

Reply:
ah okay, we're approaching string freeze day for `v124` next week (cc: @adudenamedruby), so should we expect a PR to merge it in prior to that even if feature is not complete?

Reply:
For login autofill will be tough since we still don't even have a design from ux.

Reply:
ok np, as long as its communicated that translations may not be available for it should be okay!